lib/Backend/IRBuilder.cpp

```diff
@@ -3823,7 +3823,7 @@ IRBuilder::BuildElementSlotI2(Js::OpCode newOpcode, uint32 offset, Js::RegSlot r
         case Js::OpCode::LdModuleSlot:
         case Js::OpCode::StModuleSlot:
         {
-            Js::Var* moduleExportVarArrayAddr = Js::JavascriptOperators::OP_GetModuleExportSlotArrayAddress(slotId1, slotId2, m_func->GetScriptContextInfo());
+            Field(Js::Var)* moduleExportVarArrayAddr = Js::JavascriptOperators::OP_GetModuleExportSlotArrayAddress(slotId1, slotId2, m_func->GetScriptContextInfo());
```

Comment:
ah, I see, you changed the return type. however I think we should avoid using Field() in backend code. may be create a version of OP_GetModuleExportSlotArrayAddress_Backend, or force cast to Js::Var*